iphone/Classes/TiContactsPerson.m

```diff
@@ -84,33 +84,26 @@ -(NSDictionary*)getiOS9ContactProperties: (CNMutableContact*) contact
 	if (contact == nil) {
 		return nil;
 	}
-	return [[NSDictionary alloc] initWithObjectsAndKeys:contact.givenName, @"firstName",
-		contact.familyName, @"lastName",
-		contact.middleName, @"middleName",
-		contact.namePrefix, @"prefix",
-		contact.nameSuffix, @"suffix",
-		contact.nickname, @"nickname",
-		contact.phoneticGivenName, @"firstPhonetic",
-		contact.phoneticFamilyName, @"lastPhonetic",
-		contact.phoneticMiddleName, @"middlePhonetic",
-		contact.organizationName, @"organization",
-		contact.jobTitle, @"jobTitle",
-		contact.departmentName, @"department",
-		contact.emailAddresses, @"email",
-		contact.note, @"note",
-//		contact.birthday, @"birthday", birthday property has some problems in iOS9 Beta, have to be exclusively handled
```

Comment:
There was a special handle case for birthday here. Not sure if Apple has already fixed this since Beta, but it's best to revisit it. Right now, when you do showContacts and select birthday, it returns undefined. 
Please also check alternate birthday to make sure it works correctly. as well as anniversary.
For reference, the behavior should be the same as when run on iOS8.4